### PR TITLE
Polish translation of getmonero.org

### PR DESCRIPTION
**Pull request by zofiazinha**
This is the polish version of getmonero.org files.

### CODE REVIEW

**Comment by hrumag** [comment]
Ping @erciccione.

**Comment by erciccione** [comment]
Thanks for the ping @hrumag 

@zofiazinha looks like your repository is behind the master branch by many commits. You need to rebase this PR. Are you able to do it by yourself? If not contact me, we can work on this together

**Comment by zofiazinha** [comment]
Hi erciccione,

Could you send me instructions, please? I'm not very familiar with GitHub.

Thanks.


Em Sáb, 31 de mar de 2018 09:28, erciccione <notifications@github.com>
escreveu:

**Comment by erciccione** [comment]
To test this PR i had to rebase and fix the conflicts, if you add me as a collaborator to your repo i can push the changes directly here and squash allthose commits in just one, but if you wish, we can go trhough git's (github has only limited functions) workflow together, but it will need some time, so it really depends on you

**Comment by zofiazinha** [comment]
I added you as a collaborator. Please let me know if there is anything else
I should do.

Em Sáb, 31 de mar de 2018 10:25, erciccione <notifications@github.com>
escreveu:

File: _i18n/pl/index.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+                <!-- end left one-third block-->
+                
+                <!-- center one-third block-->
+                <div class="center one-third col-lg-4 col-md-4 col-sm-12 col-xs-12">
+                    <div class="info-block">
+                        <div class="row center-xs">
+                            <div class="col">
+                                <h2>Przewodnik dla użytkowników</h2>
+                                <p class="main-resources">Przewodniki do Monero krok po kroku, podzielone na kategorie. Zawiera wszystko, począwszy od zakładania portfela, do wspomagania sieci, a nawet do tego, jak edytować tę stronę.</p>
+                                <p><a href="{{site.baseurl}}/resources/user-guides/" class="btn-link btn-auto">Przejdź do przewodnika dla użytkowników</a></p>
```
Comment (erciccione) [inline _i18n/pl/index.md]: This text is too long end it end up outside the button breaking the format, please edit with something shorter

**Comment by erciccione** [comment]
ok, i admit that squashing those 240 commits wasn't fun, but now it's cleaner :P 

Only one small fix needed, then wthis PR wil be ready

**Comment by zofiazinha** [comment]
Hi,

I rewrote the text that describes User Guides and it has now 157 characters
instead of 176. Is it okay?

**Comment by erciccione** [comment]
I'm afraid no, this is how it looks at the moment
pl

**Comment by erciccione** [comment]
Oh i'm sorry, i just realized i reviewed the wrong line

File: _i18n/pl/index.md [hunk 2]
```diff
@@ -162,7 +162,7 @@
                         <div class="row center-xs">
                             <div class="col">
                                 <h2>Przewodnik dla użytkowników</h2>
-                                <p class="main-resources">Przewodniki do Monero krok po kroku, podzielone na kategorie. Zawiera wszystko, począwszy od zakładania portfela, do wspomagania sieci, a nawet do tego, jak edytować tę stronę.</p>
+                                <p class="main-resources">Przewodnik Monero podzielony na kategorie. Zawiera wszystko, począwszy od zakładania portfela, do wspomagania sieci, a nawet do tego, jak edytować tę stronę.</p>
                                 <p><a href="{{site.baseurl}}/resources/user-guides/" class="btn-link btn-auto">Przejdź do przewodnika dla użytkowników</a></p>
```
Comment (erciccione) [inline _i18n/pl/index.md]: This is the line that need to be trimmed

**Comment by zofiazinha** [comment]
Ok, having a look at it made it a bit easier to understand what was needed
to change! I trimmed the title and the "Go to" line, let me know how it
looks now.

2018-03-31 15:50 GMT-03:00 erciccione <notifications@github.com>:

**Comment by zofiazinha** [comment]
Sure. I guess it will take some time, so I´ll get to work and keep you
posted.

**Comment by zofiazinha** [comment]
Hi Erciccione,

I made some tests of the Polish version and after a couple of changes
almost everything looks perfect. There is just one thing - the links of
Moneropedia send to the original entries instead of translated ones.

Please let me know what do you think.

**Comment by erciccione** [comment]
@zofiazinha I gathered some info and looks like that's a problem with the script who manage the moneropedia page. Leave it untranslated for now, hopefully we will resolve soon

**Comment by erciccione** [comment]
Hi @zofiazinha see #686

**Comment by zofiazinha** [comment]
Ok thanks.

Em Dom, 15 de abr de 2018 12:35, erciccione <notifications@github.com>
escreveu:

**Pull request merged**